app/js/actions/draftTableAction.js

```diff
@@ -1,21 +1,21 @@
 import {
-  ADD_DRAFT_ORDER_SUCCESS,
-  DELETE_DRAFT_ORDER_SUCCESS,
-  DELETE_ALL_DRAFT_ORDERS_SUCCESS,
+  ADD_DRAFT_DRUG_ORDER_SUCCESS,
+  DELETE_DRAFT_DRUG_ORDER_SUCCESS,
+  DELETE_ALL_DRAFT_DRUG_ORDERS_SUCCESS,
```

Comment:
so the `SUCCESS`, `FAILURE` or `REQUEST` suffixes are used to depict async actions only and seeing that this is not, then it don't make sense to associate that suffix

Reply:
It's not an async action... It's not making any request